lua/nvim-lsp-ts-utils/utils.lua

```diff
@@ -30,6 +30,19 @@ M.echo_warning = function(message)
     api.nvim_echo({ { "nvim-lsp-ts-utils: " .. message, "WarningMsg" } }, true, {})
 end
 
+M.init_options = {
```

Comment:
We could maybe add a comment noting why we have this, since it doesn't fit in with the other utilities.